src/main/java/dtos/Articles.java

```diff
@@ -0,0 +1,41 @@
+package dtos;
+
+public class Articles {
+
+    private static String titleArticle = "";
+    private static String aboutArticle = "";
+    private static String contentArticle = "";
+    private static String tagArticle = "";
```

Comment:
I'm not getting why these fields are static 🤔

Reply:
At the moment they are static so that they do not become null when accessed from different stepDefinition. but it can be improved in the following card, to have a dataManager class